Yewon/Client/src/main/MainClass.java

```diff
@@ -0,0 +1,37 @@
+package main;
+
+import java.io.IOException;
+import java.net.Socket;
+import java.net.UnknownHostException;
+import java.util.ArrayList;
+import java.util.List;
+
+import net.ReadThread;
+import view.MainFrame;
+import view.RoomFrame;
+
+public class MainClass {
+
+	public static void main(String[] args) {
+		// TODO Auto-generated method stub
+		
+		Socket socket;
+		try {
+			socket = new Socket("192.168.0.11", 9000);
+			System.out.println("접속 성공");
+
+			List<RoomFrame> rfList = new ArrayList<>();
+			MainFrame mf = new MainFrame(socket, rfList);
+			new ReadThread(socket, mf).start();
+			
+		} catch (UnknownHostException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		} catch (IOException e) {
+			// TODO Auto-generated catch block
+			e.printStackTrace();
+		}

```

Comment:
딱히 exception 처리를 하는게 아니면 아래처럼 합쳐버려도 괜찮을 것 같아요

```suggestion
		} catch (Exception e) {
			e.printStackTrace();
		}

```